README.md

```diff
@@ -1,15 +1,16 @@
-[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/bluesky/tutorials/master?urlpath=lab)
+[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/bluesky/tutorials/main)
 
 # Bluesky Tutorials
 
 This is a collection of tutorials on data acquisition and analysis using Bluesky
 and scientific Python generally. There are a couple ways to use it.
 
 * Try it in the interactive sandbox in the cloud at
-  https://mybinder.org/v2/gh/bluesky/tutorials/master?urlpath=lab.
+  https://mybinder.org/v2/gh/bluesky/tutorials/main (**recommended**).
 * Browse the content non-interactively at https://blueskyproject.io/tutorials.
+* Use Docker, running `docker run -p 8888:8888 nsls2/tutorials:latest`.
 * Download this content and run it on your local machine. This is not
-  recommended for novices. See further instructions below.
+  recommended for novices. See {doc}`INSTALLATION`.
```

Comment:
{doc}`INSTALLATION` does not render properly. Probably would be better to use `[INSTALLATION](https://github.com/bluesky/tutorials/blob/main/INSTALLATION.md)` instead?